server/db/dbManager.ts

```diff
@@ -1,35 +1,52 @@
-const { ObjectId } = require('mongodb');
-const bcrypt = require('bcrypt');
+import mongo = require('mongodb');
+import { MongoClient, ObjectId } from 'mongodb';
+import bcrypt = require('bcrypt');
 
-class PostManager {
+declare interface Post {
+	_id: mongo.ObjectID;
+	title: string;
+	body: string;
+}
+
+declare interface User {
+	_id: mongo.ObjectID;
+	username: string;
+	password: string;
+}
```

Comment:
I would suggest to create a namespace with these models and keep them within the models directory.
And actually, you don't really need `declare` because it is the same as just `interface`.